pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/HashJoinOperator.java

```diff
@@ -114,7 +116,9 @@ protected void addRowsToRightTable(List<Object[]> rows) {
         }
         continue;
       }
+      Tracing.ThreadAccountantOps.sampleAndCheckInterruptionPeriodically(rightTableSize);
```

Comment:
nit: we could just probably use _rightTable.size() (or create it in case it doesn't)